### ISSUE TITLE
SK-LARGE dataset preprocess

### ISSUE THREAD

**Issue by xwjabc** [LinearSpanNetwork/LSN#1]
After augmentation for SK-LARGE dataset, the saved file list is named as `train_pair.lst`. However, it LSN code, the name of the file list is called `train_pair_255_s_1.lst`, which is different. If I directly use the `train_pair.lst` as the dataset for LSN, it easily causes the 0-loss:

```
I0128 23:58:58.170020  1434 solver.cpp:258]     Train net output #0: dsn-ls1_loss = 0 (* 1 = 0 l
oss)
I0128 23:58:58.170030  1434 solver.cpp:258]     Train net output #1: dsn-ls2_loss = 0 (* 1 = 0 l
oss)
I0128 23:58:58.170039  1434 solver.cpp:258]     Train net output #2: dsn-ls3_loss = 0 (* 1 = 0 l
oss)
I0128 23:58:58.170048  1434 solver.cpp:258]     Train net output #3: dsn-ls4_loss = 0 (* 1 = 0 l
oss)
I0128 23:58:58.170058  1434 solver.cpp:258]     Train net output #4: dsn-ls5_loss = 0 (* 1 = 0 l
oss)
```

Thus, I wonder if we need to change the ground-truth, since in original version, the scale information is kept in ground-truth (scale = pixel value [ranged from 1 to 255]). If we directly use HED code, scale<127.5 will be eliminated, which means most of the skeleton annotations disappear. Since the file list is named as `train_pair_255_s_1.lst`, does it mean that we need to change all non-zero scales into 255 (which represents only one scale left) ?

**Comment by INGnowait** [comment]
I have the same problem as you. And I have tired to change all non-zero scales into 255. But I got a bad result. During the training, the loss value basically did not converge. So how do you solve it?

**Comment by xwjabc** [comment]
Sorry, I did not continue trying at that time.

**Comment by INGnowait** [comment]
Thank you!

**Comment by 3colorsDog** [comment]
![b718f36d4381ce909a4e799af7c1f6aa](https://user-images.githubusercontent.com/45113470/64095443-5fa34b80-cd91-11e9-82c8-bd2feafef88c.png)
same problem.
I don't change train_cal.prototxt or deploy.prototxt and get the predication by fuse = 255 * net.blobs['sigmoid-fuse'].data[0][0,:,:], then I get the terrible result, could someone help me?

**Comment by 3colorsDog** [comment]
`import numpy as np`
